bin/rucio-reaper

```diff
@@ -50,7 +51,7 @@ def get_parser():
                         help='The delay (seconds) to query replicas in BEING_DELETED state.')
     parser.add_argument('--scheme', action="store", default=None, type=str,
                         help='Force the reaper to use a particular protocol/scheme, e.g., mock')
-    parser.add_argument("--auto_exclude_threshold", "--auto-exclude-threshhold", new_option_string="--auto-exclude-threshhold", action=StoreAndDeprecateWarningAction, default=100, type=int,
+    parser.add_argument("--auto_exclude_threshold", "--auto-exclude-threshold", new_option_string="--auto-exclude-threshold", action=StoreAndDeprecateWarningAction, default=100, type=int,
```

Comment:
I think you might have misunderstood the intention here. `--auto_exclude_threshold` is already the proper option, but `--auto-exclude-threshhold` is still accepted. As such, the typo issue is already fixed, but in a backwards-compatible way.

I would say that it has been long enough that we can remove the old option, but this has to be done in a separate issue and commit that is applied in a major release.

Reply:
Reverted

Reply:
Upon revisiting this, I misspoke, but it’s best to tackle it in its own dedicated issue.